### PR TITLE
Enable ntpd service for CaaSP nodes (bsc#1108001)

### PR DESCRIPTION
**Pull request by mjura**

### CODE REVIEW

File: caasp-stack.yaml [hunk 1]
```diff
@@ -221,6 +221,16 @@ resources:
 
             suse_caasp:
               role: admin
+
+            ntp:
+              enabled: true
+              servers:
+                - ntp1.suse.de
```
Comment (rsalevsky) [inline caasp-stack.yaml]: Please don't use suse internal ntp servers in a customer heat template.

**Review by rsalevsky — CHANGES_REQUESTED** [review]

**Comment by rsalevsky** [comment]
Addional formatting of the commit message is wrong, should be "commit message (bsc#XXXXX)" and give a reason for this change.

**Comment by mjura** [comment]
@rsalevsky: done

**Comment by rsalevsky** [comment]
@mjura If you remove the comma after the node then I'm happy.

**Comment by mjura** [comment]
@rsalevsky: I'm happy too :+1:

**Review by sayalilunkad — APPROVED** [review]

**Review by rsalevsky — APPROVED** [review]

**Pull request merged**